subsys/bluetooth/audio/Kconfig.mcs

```
@@ -121,4 +121,100 @@ config BT_MCC_SHELL
 	help
 	  This option enables shell support for the Media Control Client.
 
+config BT_MCC_MINIMAL
+	bool "Minimal Media Control Client without optional procedures"
+	default n
+	help
```

```suggestion
	bool "Minimal Media Control Client without optional procedures"
	help
```

`bool` Kconfigs are `n` by default